web/mediator/manifest.js

```diff
@@ -20,8 +20,43 @@ export async function getWebAppManifest({origin, timeout = 1000}) {
 }
 
 async function _getWebAppManifest({origin}) {
-  const url = `${origin}/manifest.json`;
+  // urls are in order of priority
+  const urls = [
+    `${origin}/manifest.json`,
+    `${origin}/manifest.webmanifest`
+  ];
+  const responses = await Promise.allSettled(
+    urls.map(url => _fetchWithCache(url))
+  );
+  const response = responses.find(({value}) => value !== null);
+  return response.value || null;
```

Comment:
If `find()` returns `null` / `undefined` ... this seems like it would fail. Is it possible that every `response.value` is `null`? If so, does this solve it?:

```suggestion
  const response = responses.find(({value}) => value !== null);
  return response?.value || null;
```

Reply:
Yup! Overlooked the failing of looking up a property on null / undefined. Thanks again!